### PR TITLE
feat(Server): add get orders by filters and method

### PR DESCRIPTION
**Pull request by Mitsichury**

### CODE REVIEW

File: tools/libraries/src/Server.ts [hunk 1]
```diff
@@ -20,9 +20,10 @@ import {
   ServerOptions,
   OrderResponse,
   SupportedProtocolInfo,
-  RequestFilter,
+  RequestFilterERC20,
   SortOrder,
   SortField,
+  RequestFilterMarketPlace,
```
Comment (dmosites) [inline tools/libraries/src/Server.ts:26]: Should just be `RequestFilter` here.

File: tools/libraries/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@airswap/libraries",
-  "version": "4.0.5-beta.6",
+  "version": "4.0.5-beta.7",
```
Comment (dmosites) [inline tools/libraries/package.json:3]: Also need to update the `@airswap/types` dep version below.

**Review by dmosites — CHANGES_REQUESTED** [review]

File: tools/libraries/package.json (hunk 2 above)
Comment (Mitsichury) [reply tools/libraries/package.json:3]: done

File: tools/libraries/src/Server.ts (hunk 1 above)
Comment (Mitsichury) [reply tools/libraries/src/Server.ts]: done

**Review by dmosites — APPROVED** [review]

**Pull request merged**